packages/calcite-components/src/components/input-time-picker/input-time-picker.e2e.ts

```diff
@@ -888,6 +888,7 @@ describe("calcite-input-time-picker", () => {
       expect(await getFocusedElementProp(page, "tagName", { shadow: true })).toBe("CALCITE-INPUT-TEXT");
 
       await page.keyboard.press("ArrowDown");
+      await page.keyboard.press("Tab");
```

Comment:
This is needed now that focus doesn't automatically move to the popover when it is opened.  This is more intuitive behavior because the focus doesn't shift away without the user explicitly tabbing into it.